src/cook-web/src/app/globals.css

```
@@ -258,3 +258,164 @@ body.listen-mode-vh-fallback #root {
+  }
+
+  html.lesson-pdf-print [data-lesson-print-scroll] .content-render-table {
+    width: 100% !important;
+    table-layout: fixed;
+  }
+
+  html.lesson-pdf-print
+    [data-lesson-print-scroll]
+    :is(.content-render-th, .content-render-td) {
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)